cegs_portal/search/views/v1/dna_features.py

```diff
@@ -18,6 +18,7 @@
 DEFAULT_TABLE_LENGTH = 20
 GRCH37 = "GRCh37"
 GRCH38 = "GRCh38"
+all_assemblies = [GRCH38, GRCH37]  # Ordered by "importance"
```

Comment:
Constants are all caps

```suggestion
ALL_ASSEMBLIES = [GRCH38, GRCH37]  # Ordered by "importance"
```